### ISSUE TITLE
It doesn't work on WSL

### ISSUE THREAD

**Issue by spacepluk** [Morantron/tmux-fingers#64]
![image](https://media.giphy.com/media/cFgb5p5e1My3K/giphy.gif)

**Comment by Morantron** [comment]
steps to reproduce

![](https://media.giphy.com/media/aVytG2ds8e0tG/giphy.gif)

**Comment by spacepluk** [comment]
1. Install Windows
2. Try to use it

![](https://camo.githubusercontent.com/95f3af69c9f49951a7bd58991a45332eb24d82bb/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6346676235703565314d79334b2f67697068792e676966)

**Comment by ollyhayes** [comment]
I can't get it to work either. I installed it the same way as I did on linux but prefix - F doesn't seem to do anything where it works on my linux machine.

tmux-yank is working as expected though, so it looks like plugins are being loaded.

Here is the relevant part of my `.tmux.conf`:

```
...

set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-yank'
set -g @plugin 'Morantron/tmux-fingers'
run -b '~/.tmux/plugins/tpm/tpm'
```

I'm running ubuntu 18.04 WSL, with bash 4.4.18, tmux 2.6, gawk 4.1.4.

Is there anyway to turn on debugging or error logging or anything?
